flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/backpressure/OperatorBackPressureStats.java

```diff
@@ -25,40 +25,36 @@
 import static org.apache.flink.util.Preconditions.checkNotNull;
 
 /**
- * Back pressure statistics of multiple tasks.
- *
- * <p>Statistics are gathered by sampling stack traces of running tasks. The
- * back pressure ratio denotes the ratio of traces indicating back pressure
- * to the total number of sampled traces.
+ * Back pressure statistics of multiple tasks generated by {@link BackPressureStatsTrackerImpl}.
  */
 public class OperatorBackPressureStats implements Serializable {
 
 	private static final long serialVersionUID = 1L;
 
-	/** ID of the corresponding sample. */
-	private final int sampleId;
+	/** ID of the corresponding request. */
+	private final int requestId;
 
-	/** End time stamp of the corresponding sample. */
+	/** End time stamp when all subtask back pressure ratios were collected at the JobManager. */
```

Comment:
when all the responses of back pressure request were collected in BackPressureRequestCoordinator